### PR TITLE
`ac_radio` OK

### PR DESCRIPTION
**Pull request by Feacur**
hi there! I've chosen something fairly small for the first attempt, and here it is.

one thing that keeps eluding me is assets definitions for this particular actor; seems to be common, though, especially without any explicit symbols to base upon. on the other hand, it was quite entertaining shuffling with Z64Utils and keeping the track of files:
- [resulting af_jp.json](https://github.com/Feacur/settings/blob/main/__notes/zeldaret_af/af_jp.json)
- [exported file names](https://github.com/Feacur/settings/blob/main/__notes/zeldaret_af/file_list.txt)
- never reverified existing decompiled names, if that matters

@emilybrooks, as a final cleanup i've borrowed names from your [sheet](https://docs.google.com/spreadsheets/d/1SbJiioTu75Avsd0hRE0Hfg5-0s9BW9q2bqTYi7agUhs) for consistency. but now the list has been expanded a bit with crudely composed descriptions `__unk_*`. the lack of knowledge on the game shows itself =)

### CODE REVIEW

File: linker_scripts/jp/symbol_addrs_overlays.txt [hunk 1]
```diff
@@ -18808,21 +18808,21 @@ RO_STR_80A76860_jp = 0x80A76860; // type:asciz
...
 
 aRAD_actor_ct = 0x80A76920; // type:func
-func_80A76958_jp = 0x80A76958; // type:func
-func_80A769E4_jp = 0x80A769E4; // type:func
-func_80A76A30_jp = 0x80A76A30; // type:func
-func_80A76B2C_jp = 0x80A76B2C; // type:func
-func_80A76B4C_jp = 0x80A76B4C; // type:func
+aRAD_actor_dt = 0x80A76958; // type:func
+aRAD_set_bgOffset = 0x80A769E4; // type:func
+aRAD_animate = 0x80A76A30; // type:func
```
Comment (emilybrooks) [inline linker_scripts/jp/symbol_addrs_overlays.txt:18813]:
```suggestion
aRAD_wait = 0x80A76A30; // type:func
```

File: linker_scripts/jp/symbol_addrs_overlays.txt [hunk 2]
```diff
@@ -18808,21 +18808,21 @@ RO_STR_80A76860_jp = 0x80A76860; // type:asciz
...
 
 Radio_Profile = 0x80A76DD0; //
-D_80A76DF4_jp = 0x80A76DF4; //
-D_80A76DFC_jp = 0x80A76DFC; //
-D_80A76E10_jp = 0x80A76E10; //
-D_80A76E1C_jp = 0x80A76E1C; //
+aRAD_shadow_vtx_fix_flg_table = 0x80A76DF4; //
+aRAD_shadow_data = 0x80A76DFC; //
+aRAD_clip_offset = 0x80A76E10; //
+aRAD_processes = 0x80A76E1C; //
```
Comment (emilybrooks) [inline linker_scripts/jp/symbol_addrs_overlays.txt:18825]:
```suggestion
D_80A76E10_jp = 0x80A76E10; //
process = 0x80A76E1C; // allow_duplicated:true
```
I have no idea what `clip_offset` is supposed to be. Its not in the linker map so we don't have a definite name for it, so I would just leave it unnamed.

`process` comes from the linker map

File: src/overlays/actors/ovl_Radio/ac_radio.h [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
-#include "ultra64.h"
 #include "m_actor.h"
-#include "unk.h"
+#include "overlays/actors/ovl_Structure/ac_structure.h"
 
 struct Game_Play;
 struct Radio;
 
+typedef enum RadioProcess {
+    RADIO_PROCESS_ANIMATE,
```
Comment (emilybrooks) [inline src/overlays/actors/ovl_Radio/ac_radio.h:11]: change this to WAIT as well

**Review by emilybrooks — COMMENTED** [review]
Welcome to the project and nice job! 

We don't really have a list of rules for comments so I apologize for any confusion that may have caused. In general with comments aim for less instead of more. Comments explaining what code is doing is optional and should only be added when you've properly analyzed it, preferably checking in game as well.

**Comment by Feacur** [comment]
thank you for the [review](https://github.com/zeldaret/af/pull/226#pullrequestreview-2451775632)!

and no problem if these comments are of no use, I'll learn =)

**Review by emilybrooks — APPROVED** [review]

**Pull request merged**